packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.stories.tsx

```diff
@@ -37,11 +37,11 @@ const hyperLinkTooltip = () =>
     );
   });
 
-const Story = () => {
-  const [item] = useState({ text: new TextObject(text) });
+const Story = ({ automerge }: { automerge?: boolean }) => {
+  const [item] = useState({ text: new TextObject(text, undefined, undefined, { useAutomergeBackend: !!automerge }) });
   const model = useTextModel({ text: item.text });
   if (!model) {
-    return null;
+    return <></>;
```

Comment:
why not null?

Reply:
Storybook types didn't like null return type for component